projects/vdk-plugins/vdk-ipython/src/vdk_ipython.py

```diff
@@ -7,28 +7,55 @@
 from IPython.core.magic_arguments import argument
 from IPython.core.magic_arguments import magic_arguments
 from IPython.core.magic_arguments import parse_argstring
-from vdk.internal.builtin_plugins.run.standalone_data_job import (
-    StandaloneDataJobFactory,
-)
+from vdk.internal.builtin_plugins.run import standalone_data_job
+
+
+class JobControl:
+    """
+    The main idea of thi class is to make it easier to  work with IStandaloneDataJob instances
+    Since the user will be using this class
+    """
```

Comment:
I don't understand what `The previous version introduced a job_input that was in finalised state.` in "why" section of PR description is,
may you explain what the problem is?

Reply:
Isn't the wrapper (re)implementing the StandaloneDataJobFactory `create()`, see
https://github.com/vmware/versatile-data-kit/blob/768ec74b9dfc05b8a0207b4a4eac5955bf2e42cd/projects/vdk-core/src/vdk/internal/builtin_plugins/run/standalone_data_job.py#L196 ?

Reply:
Yes, it was but I changed the solution a little bit. U can check it again, I added one more attribute to handle calling initialise multiple times by the users

Reply:
nit: Is this a concern valid for other StandaloneDataJob usages? Seems to me, likewise jupyter notebook users, other users could also be finding themselves in a situation with initializing multiple times a StandaloneDataJob...

Reply:
yes, it is. We might raise an issue about it